astropy_helpers/conftest.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
This seems to be re-introducing pytest-specific machinery just after we've removed it to other packages.  Is that desirable?

And even if we are OK with that, there should probably be something at the top that amounts to "this is to make coverage work, *not* meant for general pytest plugin stuff"?

Reply:
Just to clarify, this isn't really a plugin, and it really is very specific to astropy-helpers, so I'll just add a comment to this effect.

Reply:
@eteq - fixed